README.md

```diff
@@ -1,2 +1,30 @@
 # hw01
 Repository for homework 01
+
+Hong Jin Jo
+====================
+
+Auto-biography
+---------------------
+
+This is a brief auto-biography of Hong Jin Jo.
+
+<img src="https://github.com/hongjinjo/myrepo/blob/master/JO_HONGJIN.jpg?raw=true">
+
+**Hong Jin Jo** was born in Seoul, Korea in 1988. He has attended the following schools:
+
+### Educational Background
+
+* Dongja Elementary School
+* Jayang Middle School
+* Korean Minjok Leadership Academy
+* Seoul National University
+
+He has acquired two sociology master degree from two institutions.
+
+1. Seoul National University
+2. The University of Chicago
+
+### Current Status
+
+Now, he is currently in his Sociology Ph.D. course at the University of Chicago. You can find more information about him at [here](https://sociology.uchicago.edu/directory/hong-jin-jo).
```

Comment:
Another suggestion is that you may do more commits to record your steps of coding. You mentioned you uploaded a photo and changed photo, but it might be better to keep track of this coding in a more logical and detailed way. Imaging when we are doing a much more complexed coding program than this one, it might be more helpful to track each crucial steps to make people understand what's going on.